doc/introduction.rst

```diff
@@ -49,19 +49,20 @@ nodes. This eliminates the need to design costly custom backplanes.
 .. _CASPER: https://casper.berkeley.edu/
 
 
+This package
+------------
 
-This module
------------
```

Comment:
There are still a few lingering references/usages of "module" rather than "package" in this file (Ctrl+F easily enough). Did you want to make a wholesale update to the new convention?

Worth mentioning `dev-guide.rst` still uses the "module" convention.